syntax = "proto3";
package organization;




import "google/protobuf/empty.proto";

/*
SetOrgExpiredDate请求
*/
message SetOrgExpiredDateRequest {
    /*
    org
    */
    string id = 1 ;
    /*
    过期日期
    */
    string expired_date = 2 ;
}

/*
SetOrgExpiredDateApi返回
*/
message SetOrgExpiredDateResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

